lib/miq_expression/field.rb

```diff
@@ -82,8 +82,20 @@ def contains(other)
     )
   end
 
+  def column_type
+    if custom_attribute_column?
+      CustomAttribute.where(:name => custom_attribute_column_name, :resource_type => model.to_s).first.try(:field_type)
```

Comment:
This will leak a `nil` out if it's not found - is that being handled by all its callers? Could there be a good reason for it not to be found?